ERDDAP_timeseries_explorer.ipynb

```diff
@@ -259,9 +259,14 @@
     "\n",
     "    standard_name = widget_std_names.value\n",
     "    widget_dsnames.value = dataset_id\n",
-    "    update_timeseries_plot(\n",
-    "        dataset=dataset_id, standard_name=standard_name, constraints=constraints\n",
-    "    )"
+    "    try:\n",
+    "        update_timeseries_plot(\n",
+    "            dataset=dataset_id, standard_name=standard_name, constraints=constraints\n",
+    "        )\n",
+    "    except:\n",
+    "        print(\n",
+    "            \"No\", standard_name, \"data for this station. Please choose another station.\"\n",
+    "        )"
```

Comment:
For some standard_names, the first dataset_id might have data, but others might not. So, when we (on the map) click on the station that does not have data, it will raise an error. I added this bit of code to ask the user to pick another station.